src/installer/installer.go

```diff
@@ -417,7 +417,14 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 	}
 	i.UpdateHostInstallProgress(models.HostStageWaitingForControlPlane, waitingForMastersStatusInfo)
 
-	if err = i.waitForMinMasterNodes(ctx, kc); err != nil {
+	cluster, callErr := i.inventoryClient.GetCluster(ctx, false)
```

Comment:
Not sure if calling back the service to get cluster details is the right way, let me know